<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hanoi</class>
 <widget class="QMainWindow" name="Hanoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hanoi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>142</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STEPS:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>142</width>
      <height>68</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <widget class="QMenu" name="menu_Set_Disks">
     <property name="title">
      <string>&amp;Set Disks</string>
     </property>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
     <addaction name="action_8"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_New_Arbitrary"/>
    <addaction name="menu_Set_Disks"/>
   </widget>
   <widget class="QMenu" name="menu_Play">
    <property name="title">
     <string>&amp;Play</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Undo_All"/>
    <addaction name="actionAuto_Play"/>
    <addaction name="actionNth_Play"/>
   </widget>
   <addaction name="menu_Game"/>
   <addaction name="menu_Play"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Undo_All">
   <property name="text">
    <string>Undo &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAuto_Play">
   <property name="text">
    <string>Auto &amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>&amp;2</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>&amp;3</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>&amp;4</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>&amp;5</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>&amp;6</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>&amp;7</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>&amp;8</string>
   </property>
  </action>
  <action name="actionNth_Play">
   <property name="text">
    <string>Nth &amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_New_Arbitrary">
   <property name="text">
    <string>&amp;New Arbitrary</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionComo_Jugar">
   <property name="text">
    <string>How Activate the Auto Player</string>
   </property>
  </action>
  <action name="actionHow_to_Play">
   <property name="text">
    <string>How to Play</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
